application/controllers/handler.js

```diff
@@ -80,7 +80,14 @@ module.exports = {
 
   login: (req, res) => {
     const query = {
-      email: new RegExp(decodeURI(req.payload.email).replace(/\s/g,''), 'i'),
+      $and: [
+        {
+          email: new RegExp(req.payload.email, 'i')
```

Comment:
use .tolowercase() instead and make sure every email in the DB is lowercase (by migrating them once and saving new emails as lowercase)